packages/expo-constants/scripts/get-app-config-android.gradle

```diff
@@ -57,9 +50,31 @@ afterEvaluate {
       }
     }
 
-    currentBundleTask.dependsOn("merge${targetName}Resources")
-    currentBundleTask.dependsOn("merge${targetName}Assets")
+    def currentAssetsCopyTask = tasks.create(
+        name: "copy${targetName}ExpoConfig",
+        type: Copy) {
+      description = "expo-constants: Copy config into ${targetName}."
+
+      into ("$buildDir/intermediates")
+      into ("assets/${targetPath}") {
+        from(assetsDir)
+      }
```

Comment:
This is what broke/was missing for 4.1.2?

Reply:
either that or https://github.com/expo/expo/pull/11926/files#diff-0e3e3ff8b908e392b3bb0385fb866db79948398c8e338079ce9077dab6b621efR68-R71

previously we were only putting the assets into one place, guessing which was correct based on the gradle version; this PR removes the need to guess and just copies them into all possible locations, so hopefully we won't be playing catch-up as much in the future as the gradle plugin seems to change back and forth.